driver/examples/snapshots/static_volsnap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: mysnapshot
```

Comment:
Hi Smita, Can we use some good names instead of mysnapshot, snap1,mysnapcontent etc